src/components/atoms/ConfirmationModal/ConfirmationModal.scss

```
@@ -1,24 +1,14 @@
 @import "scss/constants.scss";
 
 .confirmation-modal {
+  .modal-content {
+    background: $grey--modal-background;
```

Suggested name in `scss/constants.scss`:


```suggestion
    background: $modal-background--confirm;
```

---

fixed here: https://github.com/sparkletown/sparkle/pull/2131/commits/9b9f6c1a6dd9bc6c13b69121718d28b857ec4bf4